tooling/v8-snapshot/src/doctor/snapshot-doctor.ts

```diff
@@ -266,6 +266,7 @@ export class SnapshotDoctor {
   private readonly nodeEnv: string
   private readonly _scriptProcessor: AsyncScriptProcessor
   private readonly _warningsProcessor: WarningsProcessor
+  private readonly integrityCheckSource: string | undefined
```

Comment:
There seems to be a fair bit of duplication, adding this same change across multiple files and times in each file.